src/main/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/FagsakService.kt

```diff
@@ -130,7 +132,16 @@ class FagsakService(
     @Transactional
     fun lagre(fagsak: Fagsak): Fagsak {
         logger.info("${SikkerhetContext.hentSaksbehandlerNavn()} oppretter fagsak $fagsak")
-        return fagsakRepository.save(fagsak)
+        return fagsakRepository.save(fagsak).also { taskService.save(PubliserSaksstatistikkTask.lagTask(it.id)) }
+    }
+
+    @Transactional
+    fun finnOgAvsluttFagsakerSomSkalAvsluttes(): Int {
+        val fagsaker = fagsakRepository.finnFagsakerSomSkalAvsluttes()
```

Comment:
Kunne man ha bare returnert en liste med fagsaker fra `fagsakRepository.finnFagsakerSomSkalAvsluttes` istedenfor en liste med long?
Så slipper man å ta enda x antall kall mot db igjen? Blir fort en del kall dersom denne lista blir 100.

Hvis ikke så er det kanskje greit å rename `finnFagsakerSomSkalAvslutte` til `finnFagsakIderSomSkalAvsluttes` bare for å framheve at man bare får id'ne og ikke selve fagsakene.

Reply:
Tenkte på det jeg og, men hadde ikke lyst til å endre på queryen som er stjålet direkte fra ba-sak. Er litt spørsmål om hvor mye krutt man skal legge i det. 
![image](https://user-images.githubusercontent.com/17828446/206122688-fc6aa546-25e1-42da-9722-521ed5c6bb24.png)